hudi-common/src/main/avro/HoodieRollbackMetadata.avsc

```diff
@@ -51,10 +51,10 @@
      /* overlaps with 'commitsRollback' field. Adding this to track action type for all the instants being rolled back. */
      {
        "name": "instantsRollback",
-       "default": null,
+       "default": [],
        "type": {
           "type": "array",
-          "default": null,
+          "default": [],
           "items": "HoodieInstantInfo"
```

Comment:
This change is used to adapter Avro version 1.10+, which is need for Flink writer.

Reply:
Will the change affect the version less than 1.10?

Reply:
The type for field `instantsRollback` is `array` ( not a union type) which is not nullable, so it is reasonable that the default be an empty array `[]`. The old version Avro does not check the validity.

I also check the usage of method `getInstantsRollback` of class `HoodieRollbackMetadata`, the version less than 1.10 are not affected.